templates/helm/command-center/templates/argo.yaml

```diff
@@ -41,10 +41,14 @@ spec:
   targetNamespace: {{ $ns }}
   resetValues: true
   chart:
-    repository: https://broadinstitute.github.io/monster-helm
-    name: argo-controller
-    version: 0.3.1
+    git: git://github.com/broadinstitute/monster-helm.git
+    ref: dm-argo-27-rc2-DSPDC-926
+    path: charts/argo-controller
+    # repository: https://broadinstitute.github.io/monster-helm
+    # name: argo-controller
+    # version: 0.4.0
```

Comment:
Note to self: revert and point at new chart version after broadinstitute/monster-helm#12 merges.